src/content/smart-bulletins/033.md

```diff
@@ -0,0 +1,48 @@
+
+This Smart Bulletin does not address section 889-related requirements related to formal contracting procedures. Instead, it focuses on section 889 implications with regard to purchase card micro-purchases, travel and fleet card transactions. Because integrated cards combine these account types, they are not separately addressed.
+
+## Background
+
+GSA SmartPay charge cards are used to obtain over $30 billion in products and services in support of agency missions through approximately 100 million purchase, travel and fleet transactions each fiscal year. Approximately 86% of these transactions are conducted on-line. Most of these transitions are conducted by non-procurement personnel. In FY19, purchase card transactions alone totaled $22 billion, addressing a wide range of products and services. In that same year, approximately $1.2 billion of these purchases (5.7%) occurred in electronics-related Merchant Category Codes (MCCs). This information is meant to provide insight into the relative level of risk and does not capture the full scope of electronic purchases, nor does it capture purchases from merchants whose systems may use equipment/services that are now covered equipment/services. Furthermore, review of charge card purchase data does not indicate any direct purchases from the five specific sources listed above. FY19 travel card spend totaled approximately $9 billion through 44 million transactions with a limited number of airlines and rental car agencies. Hotel spend is much more widespread. There are more than 3 million travel accounts. Fleet card FY19 spend was approximately $2 billion, through almost 31 million transactions, most of which are for fuel for government vehicles. These fueling transactions take place through a large number of independent or franchised service stations.
+
+## Action
+
+The Office of Management and Budget (OMB), Office of Federal Procurement Policy (OFPP) requires agencies to develop their own policies addressing charge card compliance with section 889 (Parts A and B). Attachment 1 to this Smart Bulletin provides information, by card type, that agencies may want to consider as they develop their card use policies in relation to section 889.Agency cardholder or card manager questions regarding the application of section 889 to a particular purchase should be directed to appropriate agency acquisition, information technology, and/or supply chain risk management staff. Other questions or comments regarding this Smart Bulletin should be directed by agency headquarters A/OPCs to CCCM via email to: gsa_smartpay@gsa.gov.
```

Comment:
The Office of Management and Budget (OMB) Office of Federal Procurement Policy (OFPP) requires agencies to develop their own policies addressing charge card compliance with section 889 (Parts A and B). Attachment 1 to this Smart Bulletin provides information, by card type, that agencies may want to consider as they develop their card use policies in relation to section 889. Agency cardholder or card manager questions regarding the application of section 889 to a particular purchase should be directed to appropriate agency acquisition, information technology, and/or supply chain risk management staff. Other questions or comments regarding this Smart Bulletin should be directed by agency headquarters A/OPCs to CCCM via email to: gsa_smartpay@gsa.gov.